doc/cookbook/source/references.bib

```diff
@@ -24,3 +24,23 @@ @book{Rasmussen2005GPM
   year = {2005},
   publisher = {The MIT Press}
 }
+@article{chang2011libsvm,
+  title={LIBSVM: a library for support vector machines},
+  author={Chang, Chih-Chung and Lin, Chih-Jen},
+  journal={ACM Transactions on Intelligent Systems and Technology (TIST)},
+  volume={2},
+  number={3},
+  pages={27},
+  year={2011},
+  publisher={ACM}
+}
+@article{chang2002training,
+  title={Training v-support vector regression: theory and algorithms},
+  author={Chang, Chih-Chung and Lin, Chuan-bi},
```

Comment:
please use the same bibtex style as the other entires, i.e. abbreviate first anems